### ISSUE TITLE
Change structure to open link that has a form which on submit redirects to the password 

### ISSUE THREAD

**Issue by NinoFloris** [pglombardo/PasswordPusher#42]
I love pwpush!

But... currently views are measured through each request that comes to the given link. However a lot of businesses have mail scanners installed which crawl each link for security reasons and possibly open it. This costs you a view.

The results of this are:
- I cannot set the views to 1 as I don't know for sure if they have a mail scanner like that.
- I must therefore set the views to 2 minimum and communicate that this link has 2 views, however the issue with this is. Most people don't know that they have a link crawling mail scanner so if they get to the page and think, "oh 0 views left but it should have been 1". They don't know if it was an intercepted password or the fault of an overzealous mail scanner.

Could you fix this?

P.S. 
Something else, show ranges for the settings: this is view 1 of 3 for instance.

**Comment by pglombardo** [comment]
Hi @NinoFloris !  That's an absolutely valid issue but unfortunately there is no easy fix for that.  A view is a view whether it's by a bot or a real user.  (unless you have a suggestion I haven't thought of)

I've also hear of Skype doing similar things validating links against blacklists.

We could try to block bots by user agents but that would be spotty at best.

> P.S. 
> Something else, show ranges for the settings: this is view 1 of 3 for instance.

We do this indirectly right now correct?  Just a difference in language I think:
![screen shot 2016-08-22 at 23 26 33](https://cloud.githubusercontent.com/assets/395132/17872651/50d99ab0-68c1-11e6-9cc4-8a6f3d8c751d.png)

**Comment by pglombardo** [comment]
After re-reading this I just got what you meant for "show ranges of views".  I filed issue https://github.com/pglombardo/PasswordPusher/issues/45 to track.

One other idea that came up from [this thread](https://www.reddit.com/r/sysadmin/comments/5321q4/new_passwordpusher_features_ideas_welcome/) on reddit is the possibility of specifying a pre-shared string before a user can view a password (and eating up a view).

This may be a good work-around for the problem you described above.  Bot's wouldn't have/enter that preshared string so the true number of views is enforced.
